dracut-visible-warnings.ks.in

```diff
@@ -0,0 +1,34 @@
+#version=DEVEL
+#test name: dracut-visible-warnings
+%ksappend repos/default.ks
+
+%ksappend common/common_no_payload.ks
+%ksappend payload/default_packages.ks
+
+%post --nochroot
+SYSROOT=${ANA_INSTALL_PATH:-/mnt/sysimage}
+
+if ! [ -e "/run/anaconda/initrd_errors.txt" ]; then
+    echo "*** Failed check: Error file in the initrd wasn't created" >> $SYSROOT/root/RESULT
+fi
+
+for name in sshd vnc; do
+    ERROR_MSG="'$name' is deprecated and has been removed"
+    journalctl | grep -q "$ERROR_MSG" || echo "*** Failed check: The $name parameter warning wasn't detected in Dracut" >> $SYSROOT/root/RESULT
+
+    tmux capture-pane -p | grep -q "$ERROR_MSG" || echo "*** Failed check: The $name parameter warning wasn't detected in stage2" >> $SYSROOT/root/RESULT
+
+    REC_MSG="All usage of Anaconda boot arguments without 'inst.' prefix was removed. Please use inst.$name instead."
+
+    journalctl | grep -q "$REC_MSG" || echo "*** Failed check: The $name parameter recommendation wasn't detected in Dracut" >> $SYSROOT/root/RESULT
+
+    tmux capture-pane -p | grep -q "$REC_MSG" || echo "*** Failed check: The $name parameter recommendation wasn't detected in stage2" >> $SYSROOT/root/RESULT
+done
```

Comment:
Remove for loop and test only `sshd` or `vnc`... What do you think, is it meaningful to test two messages to know that there is not message loss?

Reply:
I've decided to test both commands. It's not making the test so heavy and could find a bug.